calendarsubscription/store.go

```diff
@@ -188,9 +189,16 @@ func (s *Store) CreateTx(ctx context.Context, tx *sql.Tx, cs *CalendarSubscripti
 		return nil, err
 	}
 
+	var now time.Time
+	err = s.now.QueryRowContext(ctx).Scan(&now)
```

Comment:
We can't issue a new query while scanning; if we're in a transaction, we need this query to happen within the transaction, and since we're scanning we can't issue a new one till it's done reading.

Instead, we can pass IssuedAt as a parameter in the auth query so we don't need to pull anything out.